_config.yml

```diff
@@ -149,4 +149,4 @@ sharing_links: # Appear at the bottom of single blog posts, add as desired. The
 
 # Site header notification content
 notification: |
-  [JuMP-dev 2020 registration delayed due to COVID-19](/meetings/louvain2020).
+  [JuMP-dev 2020 cancelled due to ongoing travel restrictions.](/jump-dev/2020/04/14/jump_dev_2020_cancellation/)
```

Comment:
due to COVID-19, not just the travel restrictions. Travel restrictions were a sufficient reason to cancel, but there are also many others like risks to participants' health. Same comment for other files.